operational/aws/long_running_instances/long_running_instances.pt

```diff
@@ -44,6 +44,13 @@ parameter "param_exclude_tags" do
   allowed_pattern /(^$)|([\w]?)+\:([\w]?)+/
 end
 
+parameter "param_automatic_action" do
+  type "list"
+  label "Automatic Actions"
+  description "When this value is set, this policy will automatically take the selected action(s)"
+  allowed_values ["Stop/Terminate Instances"]
```

Comment:
there should be two values, stop and terminate.

Reply:
Done!

Reply:
Done!